src/plone/rest/errors.py

```diff
@@ -57,7 +58,10 @@ def __call__(self):
 
     def render_exception(self, exception):
         name = type(exception).__name__
-        message = str(exception)
+        if isinstance(exception, BadRequest):
+            message = exception.args[0]
```

Comment:
`message` it looked like this in the restapi test:
```python
[{'message': 'Required input is missing.', 'field': 'email_charset', 'error': RequiredMissing('email_charset')}]
```
Later, we tried to do `json.dumps` on it:

https://github.com/plone/plone.rest/blob/8fdc78cd1420898d6ac954b5946f725ab63b33b3/src/plone/rest/errors.py#L43
As we have:

```python
'error': RequiredMissing('email_charset')
```

an error occurs:

`TypeError: Object of type RequiredMissing is not JSON serializable`

We then need to make all the keys a string.

Reply:
I fixed this. Now the message is serializable.